files/zh-cn/web/html/element/input/index.html

```diff
@@ -84,56 +84,70 @@ <h2 id="&lt;input>_types"><code>&lt;input&gt;</code> types</h2>
    <td>没有默认行为的按钮，上面显示 <a href="#attr-value">value</a> 属性的值，默认为空。</td>
    <td id="examplebutton">
     <pre class="brush: html hidden">
-&lt;input  type="button" name="button" /&gt;</pre>
-    {{EmbedLiveSample("examplebutton",200,55,"","", "nobutton")}}</td>
+      &lt;input  type="button" name="button" /&gt;
+    </pre>
```

Comment:
The `<pre>` block will be rendered as it is (except for escaping), and this is not necessary here. If you really want to do this, please follow:

```suggestion
&lt;input  type="button" name="button" /&gt;
    </pre>
```